templates/zerver/help/include/how-to-invite-users-to-join.md

```diff
@@ -7,20 +7,23 @@
    providers like Google, GitHub, GitLab, and Apple. Users can [log in with any
    allowed authentication method][logging-in], regardless of how they signed up.
 
-2. [Invite users](/help/invite-new-users) by [sending out an email or
-   sharing a reusable invite
-   link](/help/invite-new-users#send-invitations). Note that on most
-   Zulip servers (including Zulip Cloud), email invitations and
-   reusable invitation links expire 10 days after they are sent.
+1. Invite users by [sending out an email][email-invitations] or
```

Comment:
This is preexisting, but I think "sending out an email" sounds like you're doing that from your email client, which isn't right.

Reply:
How about "sending email invitations" ?

Reply:
Yeah, that works.

Reply:
Applied in my branch.